MerlinAU.sh

```diff
@@ -5176,7 +5179,12 @@ _ShowAdvancedOptionsMenu_()
    if _CheckEMailConfigFileFromAMTM_ 0
    then
        # F/W Update Email Notifications #
-       printf "\n ${GRNct}em${NOct}.  Toggle F/W Update Email Notifications"
+       if "$inRouterSWmode" 
+       then
+           printf "\n ${GRNct}em${NOct}.  Toggle F/W Update Email Notifications"
+       else
+           printf "\n ${GRNct}em${NOct}.  Toggle F/W Email Notifications"
+       fi
```

Comment:
Yes, this is much better to differentiate between the AiMesh nodes and the primary router.